package.json

```diff
@@ -30,6 +30,8 @@
     "build": "babel ./src --out-dir ./dist --copy-files",
     "build:clean": "rimraf ./build",
     "build:watch": "babel ./src --out-dir ./dist --watch",
+    "examples:build" : "babel ./examples --out-dir ./dist/examples --ignore node_modules",
```

Comment:
I WuehShould the examples at least compile...

i had a lot of headeaches to bring the pseudocode to life :/